app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -53,7 +53,9 @@ $pagination-active-bg: $sul-pagination-active-bg;
 $pagination-active-border: $sul-pagination-active-bg;
 
 // Tables
-$table-accent-bg: $light-sandstone; // background for .table-striped
+$table-cell-padding: 0.5rem;
```

Comment:
Is this a bootstrap variable? Is this going to change the padding of every table in Searchworks (e.g. course reserves, _index_locations.html.erb)?

Reply:
@cbeer yes, this is a bs variable. It changes every table. I asked @dbranchini about this and she agreed that this should be a site-wide change for consistency.